### PR TITLE
add the queries used to generate the mechelen example data

### PR DESCRIPTION
**Pull request by Rahien**

### CODE REVIEW

File: queries/mechelen-example/kieslijsten.sparql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	?kieslijst mandaat:behoortTot ?verkiezing.
+	?kieslijst ?kiesp ?kieso.
+
+	?resultaat mandaat:isResultaatVoor ?kieslijst.
+	# ?resultaat ?resp ?reso.
+
+	?resultaat mandaat:isResultaatVan ?person.
+
+} limit 100000
```
Comment (bfidlers) [inline queries/mechelen-example/kieslijsten.sparql:35]: why is this limited and the rest not?

File: queries/mechelen-example/kieslijsten.sparql
[the remark this answers is not in the document]
Comment (Rahien) [reply queries/mechelen-example/kieslijsten.sparql:1]: correct, the queries were run on prod data (anonymized)

File: queries/mechelen-example/kieslijsten.sparql (hunk 1 above)
Comment (Rahien) [reply queries/mechelen-example/kieslijsten.sparql:35]: fixing!

File: queries/mechelen-example/beleidsdomeincode.sparql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+CONSTRUCT {
+	?s ?p ?o.
+}
+WHERE {
+	?s a skos:Concept.
+	?s skos:topConceptOf <http://data.vlaanderen.be/id/conceptscheme/BeleidsdomeinCode>.
```
Comment (ruben-bottu) [inline queries/mechelen-example/beleidsdomeincode.sparql:11]: Might want to specify skos prefix?

**Review by bfidlers — APPROVED** [review]

**Pull request merged**